extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCoreMemory.cs

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Configuration;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.KernelMemory.AI;
+using Microsoft.KernelMemory.Diagnostics;
+using Microsoft.KernelMemory.MemoryStorage;
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+namespace Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoVCore;
+
+/// <summary>
+/// Azure CosmosDB Mongo vCore connector for Kernel Memory
+/// </summary>
+
+public class AzureCosmosDBMongoVCoreMemory : IMemoryDb
+{
```

Comment:
here and other classes: could you sort methods by visibility, ctor, then public, then private at the bottom?